### ISSUE TITLE
Ore drilling plant mine ore only in 8 blocks away

### ISSUE THREAD

**Issue by bm2ademon** [GTNewHorizons/GT-New-Horizons-Modpack#14279]
### Your GTNH Discord Username

bm2@ademon

### Your Pack Version

2.3.6, 2.3.0

### Your Server

sole client

### Java Version

Java 8

### Type of Server

None

### Your Expectation

Ore drilling plant should mine all ores in 48 blocks away
Setting of controller checked = 48 block
Mining pipes were decontsracted and plant was started  again. No changes.
Passive anchor was placed. Fueled for 4 hours. I leave the moon. Returned. No changes.  

### The Reality

Ore drilling plant mines only in 8 blocks away

### Your Proposal

Ore drilling plant should mine all ores in 48 blocks away

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
sounds like you used a soldering iron by mistake and disabled chunk loading

**Comment by bm2ademon** [comment]
If Im near he ore drilling plant - soldering iron does not affect. I also checked. Its not the issue.
Yes the passive anchor chould already 3x3? So again soldering iron does not affect.

**Comment by bm2ademon** [comment]
I`m also checked in my test worlds: all miners (including monoblock) mines only under pipe strictly.

**Comment by bm2ademon** [comment]
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/84045175/40fca5d0-c620-4e94-bfea-34892c995efc)

**Comment by chochem** [comment]
hmm I dont know. works fine on my end

**Comment by querns** [comment]
Use normal chunk borders, not the 3x3 ore chunk. Ore Drilling Plants preferentially mine the chunk they're in before other chunks. Also, if you placed that ore yourself, it won't mine it; the ODP only mines naturally-occurring ore.

**Comment by bm2ademon** [comment]
I suspect the problem was with "mining chunk loading" mode. (~~preferentially~~ **only** mine the chunk they're in before other chunks)
I:
- changed to no self-mining mode.
- removed 1 block (to get structure status incomplete).
- returned 1 block (to get status complete).
- turn plant on.

**Comment by KacperKuryllo** [comment]
I recently hit the same issue. By default the ore drilling plant only mines in it's own chunk, then stops. This is in a single player world with my character within close proximity to the ore miner the entire time. No amount of restarting the ore miner would fix the issue.

I was able to resolve the issue (similar to @bm2ademon ) by use of a solder iron to set the ore mining plant to "Mining chunk loading disable". Seems the chunk loading mode is the cause of this issue for me as well.

When this error occurs the mining rig would mine till bedrock (in it's own chunk) then report that mining pipes are missing (even tough they were present in both controller and input bus), and stop mining. When I disabled "Mining chunk loading", it mined in all chunks as the pipe was extracted.

I only tested this on Phobos
